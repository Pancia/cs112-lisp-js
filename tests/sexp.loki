(2 3)
(+ 2 3)
(plus 2 3)
(x 2 [] {} q)
