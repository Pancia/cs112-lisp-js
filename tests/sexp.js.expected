[List (fromList [("fileType","js")]) [Number (fromList [("fileType","js")]) 2,Number (fromList [("fileType","js")]) 3],
List (fromList [("fileType","js")]) [Atom (fromList [("fileType","js")]) "+",Number (fromList [("fileType","js")]) 2,Number (fromList [("fileType","js")]) 3],
List (fromList [("fileType","js")]) [Atom (fromList [("fileType","js")]) "plus",Number (fromList [("fileType","js")]) 2,Number (fromList [("fileType","js")]) 3],
List (fromList [("fileType","js")]) [Atom (fromList [("fileType","js")]) "x",Number (fromList [("fileType","js")]) 2,List (fromList [("fileType","js")]) [],Map (fromList [("fileType","js")]) [] [],Atom (fromList [("fileType","js")]) "q"]
]
